packages/controller/src/reducers/graphql/integrations.ts

```diff
@@ -24,46 +24,53 @@ import { ResourceCache } from "./util";
 
 import dbClient from "../../dbClient";
 
-export interface Exporter {
+export interface Integration {
+  // UUID
+  id: string;
+  // Arbitrary string provided by the user
   name: string;
-  tenant: string;
-  type: string;
-  credential?: string | null;
-  config: string;
+  // Derived from the name, more user-friendly than the id
+  key: string;
+  kind: string;
+  tenant_id: string;
+  // This data is defined on a per-integration basis, see *IntegrationData types.
+  // Some integration kinds do not involve the controller at all, so we allow this to be any.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any;
 }
-export type Exporters = Exporter[];
+export type Integrations = Integration[];
 
 export const actions = {
```

Comment:
This file feels like its a duplicate now of the state we already load for Integrations https://github.com/opstrace/opstrace/blob/main/packages/app/src/state/integration/reducer.ts

Reply:
my bad, this is the controller! Carry on :-D